### DISCUSSION TITLE
kcadm background problem

### DISCUSSION THREAD

**Discussion by bpnx** [keycloak/keycloak#16171] [Q&A]
Hello, I'm not able to send a kcadm.sh script in background. For example:
/opt/keycloak/bin/kcadm.sh get users -r MYREALM -q username='myusername' </dev/null >/tmp/output 2>/dev/null &
stops immediately. If I bring it back in foreground it works as expected.
Is there a special way to call kcadm.sh in order to use it in a background script ?
Keycloak version: 11.0.3, on linux
Thank you
M.

**Comment by wugy0103** [comment]
这是来自QQ邮箱的自动回复邮件。
